src/main/java/chopchop/MainApp.java

```diff
@@ -56,41 +56,55 @@ public void init() throws Exception {
 
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        IngredientBookStorage ingredientBookStorage =
+            new JsonIngredientBookStorage(userPrefs.getIngredientBookFilePath());
+        RecipeBookStorage recipeBookStorage = new JsonRecipeBookStorage(userPrefs.getRecipeBookFilePath());
+        storage = new StorageManager(ingredientBookStorage, recipeBookStorage, userPrefsStorage);
 
         initLogging(config);
 
         model = initModelManager(storage, userPrefs);
 
-        logic = new LogicManager(model, storage);
+        //logic = new LogicManager(model, storage);
 
-        ui = new UiManager(logic);
-    }
+        //ui = new UiManager(logic);
```

Comment:
Wouldn't this cause the logic and ui to not work?

Reply:
I commented it out cuz LogicManager and UiManager both ain't in chopchop yet. So the red line kind of annoying.